cmd/thermal-recorder/cptv_test.go

```
@@ -0,0 +1,150 @@
+// Copyright 2018 The Cacophony Project. All rights reserved.
+// Use of this source code is governed by the Apache License Version 2.0;
+// see the LICENSE file for further details.
+
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func CurrentConfig() *Config {
+	dir := GetBaseDir()
+	config_file := strings.Replace(dir, "cmd/thermal-recorder", "thermal-recorder-TEMPLATE.yaml", 1)
+	uploader_config_file := dir + "/motiontest/thermal-uploader-test.yaml"
+	config, _ := ParseConfigFiles(config_file, uploader_config_file)
```

You shouldn't ignore any error that comes back here. Given that it's test code is perfectly OK to panic here.